src/main/java/org/folio/circulation/domain/notice/schedule/ScheduledNotice.java

```diff
@@ -3,44 +3,43 @@
 import org.joda.time.DateTime;
 
 public class ScheduledNotice {
-
   private final String id;
-  private final String loanId;
-  private final String requestId;
-  private final String recipientUserId;
   private final TriggeringEvent triggeringEvent;
   private final DateTime nextRunTime;
   private final ScheduledNoticeConfig configuration;
+  private final ReferencedIds referencedIds;
+
   public ScheduledNotice(String id,
-                         String loanId,
-                         String requestId,
-                         String recipientUserId,
-                         TriggeringEvent triggeringEvent,
-                         DateTime nextRunTime,
-                         ScheduledNoticeConfig configuration) {
+    TriggeringEvent triggeringEvent,
+    DateTime nextRunTime,
+    ScheduledNoticeConfig configuration,
+    ReferencedIds referencedIds) {
```

Comment:
Can we use standard spacing here instead of one-parameter-per-line?

Reply:
Done.